Examples/Tests/quartz_vs_pec_comparison/README_physical_verification.md

```diff
@@ -0,0 +1,132 @@
+# 石英边界条件物理验证指南
```

Comment:
Hi @Miracle-Ein, appreciate the readme!

Can you please add all documentation in English for this project? That is the language all our international contributors use.